remote-wallet/src/lib.rs

```diff
@@ -1,7 +1,14 @@
 #![allow(clippy::arithmetic_side_effects)]
 #![allow(dead_code)]
-pub mod ledger;
-pub mod ledger_error;
+
+pub mod errors;
 pub mod locator;
 pub mod remote_keypair;
 pub mod remote_wallet;
+pub mod transport;
+pub mod wallet;
+
+pub trait Transport: Send {
+    fn write(&self, data: &[u8]) -> Result<usize, String>;
+    fn read(&self) -> Result<Vec<u8>, String>;
+}
```

Comment:
I don't see the utility of this trait considering we already have `RemoteWallet` -- how about keeping it simple and only having that?